### DISCUSSION TITLE
How to Docker swarm example for me.

### DISCUSSION THREAD

**Discussion by dudkinox** [community/community#42034] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

How to Docker swarm example for me.

**Comment by HEEPOKE** [comment]
You will read Document

https://docs.docker.com/engine/swarm/swarm-tutorial/

**Comment by getlost01** [comment, marked as answer]
Refer to this article for better knowledge of docker swarm with examples.
https://www.simplilearn.com/tutorials/docker-tutorial/docker-swarm
